service/debugger/debugger.go

```diff
@@ -98,6 +108,15 @@ type Config struct {
 	// TTY is passed along to the target process on creation. Used to specify a
 	// TTY for that process.
 	TTY string
+
+	// Packages contains the packages that we are debugging.
+	Packages []string
+
+	// BuildFlags contains the flags passed to the compiler.
+	BuildFlags string
+
+	// ExecuteKind contains the kind of the executed program.
+	Kind ExecuteKind
```

Comment:
nit: `Kind` is a little vague, maybe we stutter a bit and name this field `ExecuteKind` as well.

Reply:
Done. `ExecuteKind` sounds perfect. I updated a few more places where it appears.